smdebug/core/locations.py

```diff
@@ -105,6 +106,15 @@ def get_file_location(self, base_dir=""):
         return os.path.join(event_key_prefix, self.get_filename())
 
 
+class TraceFileLocation:
+    def __init__(self, mode=None):
+        self.mode = mode
+
+    @property
+    def get_file_location(self):
+        return os.getenv(SM_PROFILER_FILE_PATH_ENV_STR, DEFAULT_SAGEMAKER_PROFILER_PATH)
```

Comment:
get file location return is profiler config json?

Reply:
changed to new file path format